### DISCUSSION TITLE
SSL/TLS for Database Connections

### DISCUSSION THREAD

**Discussion by blairliikala** [ExpressionEngine/ExpressionEngine#3427] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**
Trying to connect to remote databases that require the use of SSL/TLS.

**Describe the solution you'd like**
Enable SSL/TLS connections for databases.

Currently this is the error:
![image](https://github.com/ExpressionEngine/ExpressionEngine/assets/4128963/e9d53f84-c238-4da4-a953-e269f31d0d82)

**Comment by intoeetive** [comment]
Aiming to get this resolved in https://github.com/ExpressionEngine/ExpressionEngine/pull/3716/files
